naughty-file.md

```diff
@@ -3,3 +3,6 @@ Probably don't add files on GitHub if you're working on a C# project in a team.
 ### Demo purposes
 
 Making another commit to show that it turns up in the PR
+
```

Comment:
Remove white space

Reply:
I have read and made changes according to the feedback - this will involve making more commits.